v21.2/set-locality.md

```diff
@@ -36,6 +36,10 @@ The user must be a member of the [`admin`](authorization.html#roles) or [owner](
 
 ## Examples
 
+{{site.data.alerts.callout_info}}
+[`RESTORE`](restore.html) on [`REGIONAL BY TABLE`](#regional-by-table) and [`GLOBAL`](#global) tables is supported with some limitations — see [Restoring to multi-region databases](restore.html#restoring-to-multi-region-databases) for more detail. Tables set to a [`REGIONAL BY ROW`](#regional-by-row) table locality cannot be restored.
```

Comment:
I think it's worth breaking these sentences with a newline between them since one thing is saying "you can" and one is saying "you cannot"

Also think it's worth adding a mention and link to the full Known Limitations page section from  the "Tables set to RBR cannot be restored" part